locales-src/asrouter.ftl

```diff
@@ -81,3 +81,25 @@ cfr-whatsnew-button =
   .tooltiptext = What’s New
 
 cfr-whatsnew-panel-header = What’s New
+
+## Bookmark Sync
+
+cfr-sync-bookmarks-type = Recommended Feature
```

Comment:
I think this should be one string used for all this type of recommendations, and not repeated for each one of them?

Reply:
Done! I've created a cfr-feature-type-recommended variable just above the section.

Reply:
Please make sure to add a comment to explain in which content the string is used.